### ISSUE TITLE
PluginCommandStub.Commands.EditorGetRect(out Rectangle wndRect) seems to always return Rect(X=0, Y=0, Width=400, Height=300)

### ISSUE THREAD

**Issue by paulpv** [obiwanjacobi/vst.net#63]
**Describe the bug**
I am so far only using Jacobi.Vst.Samples.Host with https://www.reaper.fm/reaplugs/.
Whenever I show any of their Editors, the returned Rect size is always 400x300.
![image](https://user-images.githubusercontent.com/1393897/186280589-9732a21a-75e5-43f6-86d7-2290b55aec4c.png)

**To Reproduce**
1. Install https://www.reaper.fm/reaplugs/
2. Launch Jacobi.Vst.Samples.Host, load any of the plugins
3. Set a breakpoint at EditorFrame.ShowDialog call to PluginCommandStub.Commands.EditorGetRect
4. View the plugin's Editor

**Expected behavior**
The plugin returns the correct Rect size.
I have stepped through the code in OBS Studio and it loads the correct Rect size from all reaplugs.

**Actual behavior**
Rect size always comes back as `Rect(X=0, Y=0, Width=400, Height=300)`

**Desktop:**
 - OS: Windows 10 Pro 21H1 19043.1889

**Comment by paulpv** [comment]
See https://github.com/NightVsKnight/vst.net/commit/4d69bebdd77413584f841d66592e8e1b6095e596

It returns non-400x300 rect when queried in OnLoad, which appears to be the more appropriate place to resize the control.

**Comment by obiwanjacobi** [comment]
My guess is that this plugin just returns 400x300 as a default value until it has initialized fully.
I have enhanced the Host Sample to implement a more robust Window Size handling of the plugin editor window.
The `EditorGetRect` call after the plugin has initialize yield more sensible values.
I have also forwarded the `SizeWindow` call on the host command stub to the `EditorFrame`. This would allow dynamically sizing plugin editors to adjust the containing editor window.
